apps/prairielearn/src/ee/lib/billing/plan-grants.ts

```diff
@@ -46,3 +57,45 @@ export async function checkPlanGrants(res: Response) {
 
   return planGrantsMatchPlanFeatures(planGrantNames, requiredPlans);
 }
+
+export async function checkPlanGrantsForQuestion(res: Response) {
+  if (userHasRole(res)) {
+    return true;
+  }
+
+  // For the time being, put this being a feature flag so that we can land
+  // this change before we've created plan grants for all existing institutions.
+  const shouldCheck = await features.enabledFromLocals(
+    'enforce-plan-grants-for-questions',
```

Comment:
Bikeshed on name? 🤷

Reply:
It's fine with me -- it's only temporary anyway.